deploy-service/teletraanservice/src/main/java/com/pinterest/teletraan/security/TokenAuthFilter.java

```diff
@@ -66,7 +66,8 @@ public void filter(ContainerRequestContext context) throws IOException {
     private SecurityContext authenticate(ContainerRequestContext context) throws Exception {
         String authCredentials = context.getHeaderString(AUTHENTICATION_HEADER);
         if (StringUtils.isEmpty(authCredentials)) {
-            throw new DeployInternalException("Can not find HTTP header: Authorization!");
+            return userDataHelper.getUserSecurityContext(null);
+            //throw new DeployInternalException("Can not find HTTP header: Authorization!");
```

Comment:
just for testing. I don't think we should commit this file. It basically bypasses authentication. 
Authorization could fail though.